### PR TITLE
Aw html form

### PR DESCRIPTION
**Pull request by allisonjw**
#### What's this PR do?
- add HTML forms for posting data for activity, hydration and sleep
- add some styling to see forms
- adds accessibility in forms

#### Where should the reviewer start?
- html form and scss

#### How should this be manually tested?
- open UI/UX and see the forms

#### Questions:
- we will need to add tabindex attributes to the form still. 
- A screen reader should be ran to see our other accessibility opportunities 
@trishalanglois @cammac60

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -46,6 +44,12 @@ body {
   width: 45%;
 }
 
+%buttonHover {
```
Comment (trishalanglois) [inline src/css/base.scss:47]: Love the styling of the buttons!

File: src/css/base.scss [hunk 2]
```diff
@@ -46,6 +44,12 @@ body {
...
+  cursor: pointer;
```
Comment (trishalanglois) [inline src/css/base.scss:48]: The pointer cursor is great.  This will help with accessibility, too.

File: src/index.html [hunk 3]
```diff
@@ -34,6 +34,55 @@ <h1>FitLit</h1>
...
+            <p class="form__hydration--error" hidden>Enter a Valid Date</p>
+            <input aria-required="true" class="form-control invalid" type="text" name="date" pattern="" placeholder="YYYY/DD/MM">
+            <label>Ounces of Water Drank</label>
+            <p class="form__hydration--error" hidden>Enter a Valid Number</p>
+            <input aria-required="true" class="form-control invalid" type="number" name="number" pattern="" min="1" max="2" placeholder="##">
+            <button type="submit">Submit</button>
+          </form>
+      </article>
+      <article class="addSleep__article">
+        <form method="POST" hidden>
```
Comment (trishalanglois) [inline src/index.html]: I know I mentioned this when you showed us earlier, but the accessibility on the forms is fantastic.  Nice work!

**Comment by allisonjw** [comment]
#### What's this PR do?
- add the step attribute to allow for decimals
- removes the pattern attribute on the date inputs because it was triggering the invalid class even though the date was correctly formatted
 - add regex pattern for decimal and numbers only.

#### Where should the reviewer start?
- same as original PR

#### QUESTIONS
- not sure if we want to keep the regex pattern on the HTML or have a function that test for it?

**Review by cammac60 — APPROVED** [review]
These are looking great! I love the idea of having different 'tabs' so that only one form is visible at a time.

**Pull request merged**